bespin-web/scripts/load-fixtures.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python manage.py loaddata ./fixtures/*.yaml /private/fixtures/*.yaml
```

Comment:
This seems fragile - hardcoding`/private/fixtures` inside this script but not referencing it elsewhere in the container image.

Since the specifics of the fixtures (what files to load and where they are located) are known to the ansible role, would it be simpler to just centralize there? We could remove this `load-fixtures.sh` script entirely and update the https://github.com/Duke-GCB/gcb-ansible-roles/pull/13/files#diff-f8eb32e09f20ee8c4382bede46ba52c6R30 line to run `"python manage.py loaddata {{ fixture_file }}"`  on a list of the fixtures